tests/linux/test_local_network_access_manager.cpp

```diff
@@ -123,7 +123,7 @@ struct LocalSocketWriteErrorTestSuite : LocalNetworkAccessManager, WithParamInte
 {
 };
 
-const std::vector<int> local_socket_write_error_suite_inputs{0, 1, 2, 3, 4, 5};
+const std::vector<int> local_socket_write_error_suite_inputs{0, 1, 2, 3, 4, 5, 6};
```

Comment:
This feels like we'll forget to update it :)

[`Range(begin, end [, step])`](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#how-to-write-value-parameterized-tests), too :)

Reply:
I agree, but it needs to match the number of `local_socket_write()` calls in `LocalSocketReply::send_request()`.  And sure, it could be a range as well, but this was quick to implement :smiley: